lib/Drupal/drupal_helpers/Field.php

```diff
@@ -19,6 +19,8 @@ class Field {
    *
    * @param string $field_name
    *   Machine name of the Field.
+   *
+   * @throws \Exception
```

Comment:
@alexdesignworks Sorry, my comment below about this `@throws` docblock was made via email, so it didn't return inline.

Here it is again inline:

This exception documentation: 

     * @throws \Exception

 is informative as it is, though I'm happy to add more information.

Uncaught exceptions in Drupal code are a big problem because most methods
that might throw exceptions do not document that correctly.

Please consult the current coding standards for Drupal and Symfony if you
are in any doubt as to the value and necessity of `@throw` in docblock
comments. It would be very difficult to add information about all the different reasons that Exceptions that may be thrown here, as what I'm attempting to do is catch exceptions thrown elsewhere, in Rules, and provide some context.

My end-goal though is to catch them and eventually move to a model where we throw a DrupalUpdateException in drupal_helpers, that will properly trigger a stop to further hook_update_N executions.